cmd/nerdctl/logs.go

```diff
@@ -167,6 +176,23 @@ func logsAction(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func getLogPath(ctx context.Context, container containerd.Container) (string, error) {
+	extensions, err := container.Extensions(ctx)
+	if err != nil {
+		return "", fmt.Errorf("get extensions for container %s,failed: %#v", container.ID(), err)
+	}
+	metaData := extensions[k8slabels.ContainerMetadataExtension]
+	var meta containerstore.Metadata
```

Comment:
Probably we just need this to avoid importing the CRI deps
```go
type CRIContainerVersionedMetadata struct {
  Version string
  Metadata CRIContainerMetadata
}
type CRIContainerMetadata struct {
  LogPath string
}
```

Reply:
looks like a good idea

Reply:
updated